OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -260,6 +267,9 @@
     <Compile Include="Traits\Power\Power.cs" />
     <Compile Include="Traits\Power\RequiresPower.cs" />
     <Compile Include="Traits\Power\ScalePowerWithHealth.cs" />
+    <Compile Include="Traits\Production.cs" />
+    <Compile Include="Traits\ProductionBar.cs" />
```

Comment:
This should not be here. I was going to suggest Render or Graphics, (was leaning toward the latter), but I see there is `TimedUpgradeBar` in `Traits\Render`, which is actually a perfect example ;)